### PR TITLE
Creating a file for San Gwann, Malta

### LINKED ISSUE #90: abcfantasy hometown file
## San Gwann, Malta
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [x] Have a ocnversation
- [x] Get a :+1:
- [x] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by abcfantasy**
This resolves #90

### CODE REVIEW

File: abcfantasy-sangwann.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## San Gwann, Malta
+
+### Places to eat
+
+
```
Comment (abcfantasy) [inline abcfantasy-sangwann.md:6]: @finoutlook why are there 2 linebreaks here?

File: abcfantasy-sangwann.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## San Gwann, Malta
```
Comment (abcfantasy) [inline abcfantasy-sangwann.md]: @KLia Ghandi bzonnok tikkoregi l-Malti!

File: abcfantasy-sangwann.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+### Places to eat
+
```
Comment (finoutlook) [inline abcfantasy-sangwann.md:4]: @abcfantasy Can you get koldskål in the danish village?

File: abcfantasy-sangwann.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+## San Gwann, Malta
+
+### Places to eat
```
Comment (jennie-r) [inline abcfantasy-sangwann.md:3]: looking forward to recommendations for San Gwann!

File: abcfantasy-sangwann.md (hunk 3 above)
Comment (abcfantasy) [inline abcfantasy-sangwann.md:4]: @finoutlook perhaps. You should check it out!

File: abcfantasy-sangwann.md (hunk 2 above)
Comment (KLia) [inline abcfantasy-sangwann.md]: jaqq!

Comment (KLia) [inline abcfantasy-sangwann.md]: @abcfantasy San Ġwann

Comment (abcfantasy) [inline abcfantasy-sangwann.md]: @KLia Grazzi!

**Comment by abcfantasy** [comment]
This looks :+1:

**Pull request merged**